src/renderer/src/hooks/useMapeoDeviceList.ts

```diff
@@ -0,0 +1,27 @@
+import { useMapeoDeviceStore } from './stores/mapeoDeviceStore'
+
+export const useMapeoDeviceMembersListIds = () => {
+  const devices = useMapeoDeviceStore(
+    (store) => store.memberDevices,
+    (oldVal, newVal) => {
+      const oldValArray = Object.keys(oldVal).map((key) => key)
+      const newValArray = Object.keys(newVal).map((key) => key)
+      return oldValArray !== newValArray
+    },
+  )
+
+  return Object.keys(devices).map(([key]) => key)
+}
+
+export const useMapeoDeviceNonMembersListIds = () => {
+  const devices = useMapeoDeviceStore(
+    (store) => store.nonMemberDevices,
+    (oldVal, newVal) => {
+      const oldValArray = Object.keys(oldVal).map((key) => key)
+      const newValArray = Object.keys(newVal).map((key) => key)
+      return oldValArray !== newValArray
```

Comment:
similar comment about this always returning true

Reply:
https://github.com/digidem/mapeo-desktop-shell/pull/40/commits/9d76998e7e969ecdb734c309a9249eed7a8b74c6